docs/testcases/UI/Pipeline_library/Pipeline_details/2445.md

```diff
@@ -0,0 +1,29 @@
+# Check Markdown for the Tools/Pipelines Description
+
+Test verifies the Markdown rendering in the Tools/Pipelines descriptions works correctly.
+
+**Prerequisites**:
+- Admin user
+- Test tool
+
+**Preparations**:
+1. Login as admin user from the prerequisites
+2. Open the **Library** page
+3. Create the pipeline from the **DEFAULT** template
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Tools** page | |
+| 2 | Select test tool | Description tool page opens | 
+| 3 | Click **Edit** button in the *Full description* section | |
+| 4 | Save the text of the test tool description, then clear and input markdown that contains: <li> bulleted list <li> numbered list <li> different levels headers <li> bold, italic and strikethrough text <li> quoting text <li> a task list <br> etc. | |
```

Comment:
I suppose to add a raw Markdown file to the case Prerequisites with the content like:
```
# Header1
## Header2
### Header3
**bold text**
_cursive text_
...
```

And during this step paste the content of that file. In the expected results, specify all results that shall be shown